extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.control.js

```diff
@@ -527,6 +527,14 @@ angular.module('PaperUI.controllers.control', []).controller('ControlPageControl
     if ($scope.item.state === 'UNDEF' || $scope.item.state === 'NULL') {
         $scope.item.state = '-';
     }
+    $scope.sendStateCommand = function(command) {
+        $scope.sendCommand(command);
+        if (command == "DOWN") {
+            $scope.item.state = 100;
```

Comment:
This is a click handler and I am just setting the state before sending the command to the runtime. Its being done the same way in all other control widgets.